### PR TITLE
rpcclient: Add GetMasterPubkey

### PR DESCRIPTION
**Pull request by JoeGruffins**
Hey. We would like to use this method in dcrstakepool.  There was a scary wall of text here...

### CODE REVIEW

**Comment by JoeGruffins** [comment]
I'll remove that 'k' on publick along with whatever else needs to be changed :/

**Comment by jrick** [comment]
This should be using hdkeychain/v2

**Comment by JoeGruffins** [comment]
@jrick Thank you.

Does this work or is there a better way to get the network params?
Not sure why travis is angry...

**Comment by JoeGruffins** [comment]
Thank you!
My next step is to rebase this, correct?

File: rpcclient/wallet.go [hunk 1]
```diff
@@ -1375,6 +1376,51 @@ func (c *Client) ListAccountsMinConf(minConfirms int) (map[string]dcrutil.Amount
...
+		return nil, err
+	}
+
+	var pubkeystr string
+	err = json.Unmarshal(res, &pubkeystr)
+	if err != nil {
+		return nil, err
+	}
+
+	// pubkey is a hierarchical deterministic master public key (BIP0032)
```
Comment (davecgh) [inline rpcclient/wallet.go]: This is incorrect.  There is no guarantee what exact scheme it's using, but the current one is actually BIP0044, not 32.  I'd just remove the reference.

Comment (JoeGruffins) [reply rpcclient/wallet.go]: Thank you. Are there any more changes that need to be made?

Comment (davecgh) [reply rpcclient/wallet.go]: Thanks.  Looks good.

**Pull request merged**